param/parameters.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import Any, Optional
+
+class Number:
+    default: float
+    def __get__(self, instance: Any, owner: Any) -> float: ...
+    def __set__(self, instance: Any, value: float) -> None: ...
+    def __init__(self, default: Optional[float] = ..., doc: Optional[str] = ...) -> None: ...
```

Comment:
Is the `__init__` required? I got more info before in VS Code:

Before:
<img width="699" height="357" alt="image" src="https://github.com/user-attachments/assets/14be0d70-25fb-4c97-b354-560dba43dc9c" />

Now:
<img width="699" height="213" alt="image" src="https://github.com/user-attachments/assets/d33b5779-4411-4bde-9ffa-e71540b1547f" />

Reply:
Regarding first question. Yes static type checkers would complain that you would have to check for `None`. But now you only have two possibilities `str | None`. To some that is a big help. The ones that do no appreciate types can just let them be.

Reply:
I agree that having type annotations would be very useful. But I also think that if we have a library that provides type annotations that are not always correct, then we should provide a solution for people to deal with that. It might be it would just be a documentation thing, stating that these annotations should only be used in an IDE and in a special mode. I don't really know to be honest but I don't find that answer to be complete and clear enough:

> The ones that do no appreciate types can just let them be.